### ISSUE TITLE
Can i have a dynamic log_stream_name and log_group_name in output based on the directory name?

### ISSUE THREAD

**Issue by prabhat2k15** [fluent/fluent-bit#5146]
i have setup flunetbit on kubernetes as Daemonset and pushing logs to cloudwatch.
i have the follwing logging architecture
/var/log/app/**website**/### w1/app.log
/var/log/app/**website**/### w2/app.log
/var/log/app/**api**/### api1/app.log
/var/log/app/**api**/### api2/app.log
/var/log/app/**db**/### db1/app.log
/var/log/app/**db**/### db2/app.log
/var/log/app/**db**/### db3/app.log
Can i have a dynamic log_stream_name and log_group_name in output based on the directory name?
What i want is to fetch my app directory name (**website,api,db**) and use it as log_group_name
and fetch my app subdirectory (### w1,w2,db1,db2,db3,api1,api2) as log_stream_name
### **_Is it possible in fluentbit?_**
Example output config
```
[OUTPUT]
        Name cloudwatch_logs
        Match *
        log_stream_name ${app_subdirectory}
        log_group_name ${app}
        region ap-southeast-1
        # log_format json/emf
        metric_namespace fluent-bit-metrics
        metric_dimensions ec2_instance_id
        auto_create_group true
```

**Comment by trallnag** [comment]
Yes. Use the prefix option instead of hardcoding the stream name. Check the documentation of the plugin.

You will be able to generate streams named something like this: "var.log.myapp.blablabla".

**Comment by lawrenceong** [comment]
> Yes. Use the prefix option instead of hardcoding the stream name. Check the documentation of the plugin.
> 
> You will be able to generate streams named something like this: "var.log.myapp.blablabla".

Unfortunately, using the prefix option does not get rid of `var.log.app`.  Combined with Kubernetes, this makes it even messier and longer.  Would prefer that the templating option is implemented instead that is already in the [go plugin](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit).
